relm4-book/src/widget_macro.md

```diff
@@ -4,7 +4,7 @@ To simplify the implementation of the `Widgets` trait, let's use the relm4-macro
 
 > An almost identical example called "simple" is available [here](https://github.com/AaronErhardt/relm4/tree/main/relm4-examples) if you want to see the code in action.
 
-## What different?
+## What's the difference?
```

Comment:
I think "What's different" would be better. What do you think?

Reply:
Fine for me.